recipe/meta.yaml

```diff
@@ -34,22 +35,24 @@ requirements:
     - jinja2
     - cython
     - setuptools_scm
+    - wheel
     - extension-helpers
-    - numpy >=1.17
+    - numpy
     - scipy >=1.1
     - pyyaml >=3.13
     - matplotlib >=3.0
   run:
     - python
-    - pyerfa
-    - numpy >=1.17
+    - pyerfa >=2.0
+    - numpy >=1.19
```

Comment:
This should be something like `{{ pin_compatible('numpy') }}` so the required version needs only to be tracked in one place (i.e., the recipe's cbc.yaml).